US National Archives Research Catalog.js

```diff
@@ -162,5 +164,5 @@ var testCases = [
 			}
 		]
 	}
-]
+];
```

Comment:
no

Reply:
I'm truly sorry about these -- I thought I had grepped out all instances. Would it be possible to merge https://github.com/zotero/translators/pull/1958? That would immediately flag them.

Reply:
Don't be sorry. It is easy to spot for me and I guess easy to fix for you.